db/schema.rb

```diff
@@ -48,5 +48,36 @@
     t.index ["form_id"], name: "index_pages_on_form_id"
   end
 
+  create_table "published_forms", force: :cascade do |t|
```

Comment:
Was chatting to @danworth  about this, he made a good point that any features we add in future might need to be propagated through this table.

I'm not sure it's necessarily a blocking issue or even a major problem, I think we'd only need to regenerate if we make changes to the schema that break backwards-compatibility, but might be worth documenting somewhere so we don't forget about it.

Reply:
Yeah, this is something @thomasiles and I have been chatting about and thinking at there is still a need for paper-trail if not anything to be able to revert to specific forms or pages.

We will very much have to be mindful of backwards compatibility and its something we kind of "dodged"  a bit when we rolled out the new answer_Settings input_types.